test/parallel/test-process-internal-binding.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
How about adding a positive assertion
```js
// Flags: --expose_internals
const internalBinding = require('internal/process').internalBinding
assert.ok(internalBinding('module_wrap'));
```

Reply:
Why? We don't export `require('internal/*')` ever to my knowledge

Reply:
For internal code coverage (that's what the flag is for). e.g.
https://github.com/nodejs/node/blob/9e8f4b3834263f0c66fb0f5483c4284a4d8ea682/test/parallel/test-internal-errors.js#L1-L15
and all the `/test/parallel/test-internal-*` files

P.S. maybe name the file `test-internal-process-binding.js`

Reply:
mmmm, I think that flag would defeat the purpose of this.

Reply:
On this I yield to the OP.